components/ui/page-header/index.tsx

```
@@ -0,0 +1,133 @@
+    showBackButton,
+    onBackPress,
+    actionButtons = [],
+    rightComponent,
+    mobileActions = "none",
+}) => {
+    const theme = useTheme();
+    const colors = Colors(theme);
+    const { xl } = useBreakpoints();
+    const router = useRouter();
+    const nav = useMyNavigation();
+    const styles = useMemo(() => useStyles(colors), [colors]);
```

Biome (2.4.4)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

(lint/correctness/useHookAtTopLevel)